web/src/views/Apply/FormQuestions.tsx

```diff
@@ -98,54 +69,16 @@ export const FormQuestions = () => {
                   Incluye cualquier link personal que tengas (Github, LinkedIn, pagina web personal, etc)
                 </p>
               </div>
-              <div className="w-80 md:w-8/12 md:-mt-5">
-                <FieldArray name="links">
-                  {({ push, remove }) => (
-                    <div>
-                      {values.links.map((link, index) => (
-                        <div className="grid grid-cols-1 space-y-2 mt-2 md:-space-y-2">
-                          <div className="flex items-center space-x-2">
-                            <Field className={styles.dynamicField} name={`links.${index}.url`}/>
-                            {
-                              index !== 0 && (
-                                <button
-                                  type="button"
-                                  className="p-1 text-gray-500 hover:text-gray-800 focus:outline-none focus:text-gray-800"
-                                  onClick={() => remove(index)}
-                                >
-                                  <Image
-                                    src={deleteIcon}
-                                    className="w-8 px-1 md:w-14 md:px-3"
-                                    alt="Delete"
-                                  />
-                                </button>
-                              )
-                            }
-                            {
-                              index === 0 && (
-                                <button type="button" onClick={() => push({ url: '' })}>
-                                  <Image
-                                    src={addIcon}
-                                    className="h-10 w-10 px-2 md:h-16 md:w-16 md:px-4"
-                                    alt="Add"
-                                  />
-                                </button>
-                              )
-                            }
-                          </div>
-                          <div>
-                            <ErrorMessage
-                              name={`links.${index}.url`}
-                              component='div'
-                              className={styles.errorMsg}
-                            />
-                          </div>
-                        </div>
-                      ))}
-                    </div>
-                  )}
-                </FieldArray>
-              </div>
+              <DynamicFieldArray name="links"/>
+               { typeof errors.links === 'string' ?(
+                <div>
+                  <ErrorMessage
+                    name="links"
+                    component="div"
+                    className={styles.errorMsg}
+                  />
+                </div>
+               ) : null}
```

Comment:
Could we include this inside the `DynamicFieldArray` component?

Reply:
done 🚀